.browserslistrc

```diff
@@ -6,7 +6,6 @@ not dead
 Chrome >= 45
 Firefox >= 38
 Edge >= 12
-Explorer >= 10
```

Comment:
Already did that here: https://github.com/twbs/bootstrap/pull/23586/commits/25a348d4037bdf95ba799b9f43ec10ec969d5666 😄

Reply:
Yeah we need to decide on the order of PRs merging.